libbeat/autodiscover/providers/kubernetes/kubernetes.go

```diff
@@ -145,28 +146,23 @@ func (p *Provider) emitEvents(pod *kubernetes.Pod, flag string, containers []*ku
 	host := pod.Status.GetPodIP()
 
 	// Do not emit events without host (container is still being configured)
```

Comment:
```suggestion
	// Do not emit events without host for containers which are not stopping (container is still being configured)
```
Extending the comment here as I had to read this line multiple times. The part that wondered me here is if should be `flag == start` or something like that to dismiss the start state but not all others. Not sure what the available flags are.

Reply:
Comments extended.

Regarding flags, we only have `start` and `stop` at the moment, but I only wanted to change the behaviour of stop, and this is why explicitly check for that.